vello_shaders/shader/fine.wgsl

```diff
@@ -987,18 +988,23 @@ fn main(
                 cmd_ix += 2u;
             }
             case CMD_BEGIN_CLIP: {
+                let is_non_isolated = (tag & NON_ISOLATED_BLENDING_FLAG) != 0;
+                var rgba_mul = vec4(0.0);
+                if is_non_isolated {
+                    rgba_mul = vec4(1.0);
+                } 
                 if clip_depth < BLEND_STACK_SPLIT {
                     for (var i = 0u; i < PIXELS_PER_THREAD; i += 1u) {
                         blend_stack[clip_depth][i] = pack4x8unorm(rgba[i]);
-                        rgba[i] = vec4(0.0);
-                    }
+                        rgba[i] *= rgba_mul;
```

Comment:
You're sure the backdrop isn't now double counted, even if it has alpha? I think that `a over a` if a is partially transparent isn't equal to a.